generate-hash-list.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+tmpfile="$(mktemp)"
+tmpold="$(mktemp)"
+
+git clone https://github.com/minetest/minetest
+cd minetest || exit
+git log --oneline origin/master > minetest.log
+
+declare -A clients
+clients["dragonfireclient"]="https://github.com/dragonfireclient/dragonfireclient"
+clients["TeranClient"]="https://github.com/Teeranu/TeranClient"
+clients["TeranClientCustomized"]="https://github.com/BotBoyM1/TeranClientCustomized"
+clients["waspsaliva"]="https://github.com/VoidCosmo/waspsaliva"
+clients["OtterClient"]="https://github.com/otterminetest/OtterClient"
+
+for client in "${!clients[@]}"; do
+	git remote add "${client}" "${clients["${client}"]}".git
+	git fetch "${client}"
```

Comment:
```suggestion
	git fetch -fn "${client}"
```
Tell to never check refs/tags no matter what's in configs and additionally force fetching refs/heads even if it seems to be wrong (skipping tags should be enough but there might be other situations where checks fail because we cut a lot of data for initial origin fetch).